ceph-docker-flake8/build/build

```diff
@@ -29,7 +29,7 @@ function check(){
 
 function main() {
     # install some of our dependencies
-    if [ "${HUDSON_URL}" = "https://jenkins.ceph.com/" ]
+    if [ "${HUDSON_URL}" = "https://2.jenkins.ceph.com/" ]
```

Comment:
Why this condition at all? Was somebody trying to build this job so it could be run locally?

Reply:
Looks like: https://github.com/ceph/ceph-build/commit/83b99762805edd3bf133ee1a2d02674b90c241c5

Reply:
Maybe change it to check if the env var is set at all?

Reply:
Yeah, I think that makes better sense. Maintaining jenkins urls in jobs will get tedious if we need to add more master nodes.